docs/rules/require-v-if-else-key.md

```diff
@@ -0,0 +1,52 @@
+---
+pageClass: rule-details
+sidebarDepth: 0
+title: vue/require-v-if-else-key
+description: require `key` with `v-if/v-else-if/v-else` directives
+---
+# vue/require-v-if-else-key
+
+> require `key` with `v-if/v-else-if/v-else` directives
+
+- :exclamation: <badge text="This rule has not been released yet." vertical="middle" type="error"> ***This rule has not been released yet.*** </badge>
+
+## :book: Rule Details
+
+This rule reports the elements which have `v-if`, `v-else-if`, and/or `v-else` and do not have a `v-bind:key` or `key`.
+
```

Comment:
Can you explain here that Vue3 doesn't require this rule? In Vue3, the key is automatically assigned.

Reply:
Demo: https://vue-next-template-explorer.netlify.app/#eyJzcmMiOiI8ZGl2IHYtaWY9XCJlcnJvclwiPlxuICBFcnJvcjoge3sgZXJyb3IgfX1cbjwvZGl2PlxuPGRpdiB2LWVsc2U+XG4gIHt7IHJlc3VsdHMgfX1cbjwvZGl2PlxuIiwic3NyIjpmYWxzZSwib3B0aW9ucyI6eyJvcHRpbWl6ZUltcG9ydHMiOmZhbHNlLCJvcHRpbWl6ZUJpbmRpbmdzIjpmYWxzZX19